installer/scripts/omsadmin.sh

```diff
@@ -573,16 +614,19 @@ onboard()
             # This is a temp solution since the DSC doesn't support multi-homing now
             # Only the primary workspace receives the configuration from the DSC service
             if [ "$USER_ID" -eq "0" ]; then
-                su - $AGENT_USER -c $METACONFIG_PY > /dev/null
+                su - $AGENT_USER -c $METACONFIG_PY > /dev/null || error=$?
```

Comment:
@jeffaco Do you have any idea why this error check wasn't present before? As far as I can tell, that's been the case for a while.